doc/man/8/rbd.rst

```diff
@@ -268,6 +268,22 @@ Commands
   The --options argument is a comma separated list of device type
   specific options (opt1,opt2=val,...).
 
+:command:`device attach` [-t | --device-type *device-type*] --device *device-path* [--read-only] [--exclusive] [--force] [-o | --options *device-options*] *image-spec* | *snap-spec*
+  Attach the specified image to the specified block device (currently only
+  `nbd` on Linux). This operation is unsafe and should be used very carefully.
```

Comment:
Please revert back to:

> This operation is unsafe and should not be normally used.

I've chosen that wording because no use case outside of upgrading CSI plugin containers has been identified.  Those who manage their mappings with the CLI (whether manually or from scripts) in an unconstrained environment should not need to use `rbd device attach`.

Reply:
OK